meson.build

```diff
@@ -5,7 +5,7 @@ project('CNDP', 'C', 'cpp',
     # Get version number from file.
     # Fallback to "more" for Windows compatibility.
     version: run_command(find_program('cat', 'more'),
-        files('VERSION')).stdout().strip(),
+        files('VERSION'), check: true).stdout().strip(),
```

Comment:
If you (are able to) bump your minimum version of Meson to 0.57.0 then you can do:

```meson
version: files('VERSION')
```
and meson will handle everything here for you. No need to try to portably find some kind of `cat` and run it as a command.

Reply:
We are trying to support Ubuntu 20.04, 21.04 and 22.04 LTS versions. We will remove 20.04 support soon.

The standard installed package versions for meson for Ubuntu versions are:
20.04 - 0.53.2
21.04 - 0.57.0
22.04 - 0.61.2

I would like to bump the minimum version but the latest version we can safely bump to is 21.04 0.57.0.
I do not think we want to force developers to install different versions then what comes with the distro release.

When we drop support for 20.04 LTS we can move to 0.57.0 and fix this one.